server/src/test/java/org/elasticsearch/repositories/blobstore/BlobStoreRepositoryTest.java

```diff
@@ -242,7 +242,7 @@ private static RepositoryData addRandomSnapshotsToRepoData(RepositoryData repoDa
             int numIndices = inclIndices ? randomIntBetween(0, 20) : 0;
             final ShardGenerations.Builder builder = ShardGenerations.builder();
             for (int j = 0; j < numIndices; j++) {
-                builder.put(new IndexId(randomAlphaOfLength(8), UUIDs.randomBase64UUID()), 0, "1");
+                builder.put(new IndexId(randomAlphaOfLength(8), UUIDs.randomBase64UUID()), 0, "new");
```

Comment:
Had to change this, because of: https://github.com/crate/crate/commit/998e6bd043b35a81b4721d8214796b74b2e7c116
Otherwise the shard generations are different in this assertion: https://github.com/crate/crate/pull/13093/files#diff-8140773b36ca5f4829325e9ea4a63256b256cec6a4544dc93160f5309642f4d2R183